gatling-maven-plugin/pom.xml

```diff
@@ -99,4 +101,4 @@
 			</plugin>
 		</plugins>
 	</build>
-</project>
+</project>
```

Comment:
Please restore last empty line so git is happy. Basically, you can revert all your changes on this file.